bundle/Resources/views/themes/ngadmin/content/location_view.html.twig

```diff
@@ -26,6 +26,10 @@
                     </svg>
                 </a>
                 <a href="{{ url }}">{{ content_type.name }}</a>
+                {% set locale_conversion_map = ibexa.configResolver.getParameter('locale.conversion_map', 'ngsite') %}
+                {% for language in content.versionInfo.languages %}
+                    &nbsp {{ locale_conversion_map[language.languageCode]|split('_')|last|nglayouts_country_flag|raw }}
```

Comment:
`&nbsp` is an error here I guess? Why not just separate languages with commas?

Also, don't use `nglayouts_country_flag`, this package cannot depend on Netgen Layouts.

Reply:
>Also, don't use nglayouts_country_flag, this package cannot depend on Netgen Layouts. If needed, implement a custom Twig function to fetch the language name with LanguageService from Ibexa.

Komentar na povezanom tasku na fini glasi: 
"pliz ne zaboravi dodat razmak između content tip linka i ovih zastavica. Na Fini je  nabijeno"

Reply:
> > @iherak suggested we put this in media-site instead of here. Your thoughts?
> 
> Not ideal. Only new websites will then benefit from that unless we proactively implement it on older sites. Upgradablilty is also a problem. The template is not completely trivial, so any change here later will need to be propagated manually.

@emodric Do you have a suggestion then?

Reply:
@emodric  Is this ok now?